Zend/zend_alloc.c

```diff
@@ -69,6 +69,7 @@
 # include "win32/winutil.h"
 #endif
 
+#include <stdbool.h>
```

Comment:
This is unnecessary, stdbool.h is included centrally in zend_types.h.

Reply:
Thanks, removed it.